pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Quinn Milionis <quinn@scoutapm.com>"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.9"
```

Comment:
what's the reason for dropping version?

Reply:
I wanted to bring this in line with the python versions supported by the existing scout apm logs agent. https://github.com/scoutapp/scout_apm_python/blob/master/setup.py#L51
I plan to bump this to 3.10 in a future PR though as it helps some dependency compatibly. Also using [this](https://devguide.python.org/versions/) for a reference point.